docs/twig-components/method/first_related_content.md

```diff
@@ -1,6 +1,5 @@
 # first_related_content
 
-`first_related_content(content, name = null, publishedOnly = true)` is a Twig function to ...
+`first_related_content(content, name = null, publishedOnly = true)` ...
 
-
-Source: Bolt
\ No newline at end of file
+See [related_first filter](https://docs.bolt.cm/4.0/twig-components/functions#related-first-name-null-contenttype-null-bidirectional-true-publishedonly-true).
```

Comment:
Since this PR is going to be merged into 5.0, links should be updated to link to 5.0.